app/gui2/src/components/GraphNode.vue

```diff
@@ -253,20 +254,20 @@ function handleClick(e: PointerEvent) {
 
 <template>
   <div
-    class="Node"
     ref="rootNode"
+    class="Node"
     :style="{ transform }"
-    v-on="dragPointer.events"
     :class="{ dragging: dragPointer.dragging }"
+    v-on="dragPointer.events"
```

Comment:
can probably use `@` instead of `v-on`?

Reply:
I'm assigning multiple events declared in the `usePointer` all at once here, and I want to avoid having to specify each of them. `v-on` assignment allows that.

Reply:
ah, i guess i wasn't clear enough - i meant `@="dragPointer.events"`

Reply:
I didn't know that this syntax works. The linter complains about it, but it compiles.
![image](https://github.com/enso-org/enso/assets/919491/6357e96a-2740-46f9-8b8f-f50ef601e650)

I don't see a way to reconfigure that lint though, so I think I'll stick with `v-on`.
https://eslint.vuejs.org/rules/valid-attribute-name.html